### DISCUSSION TITLE
RangeSlider - Different Track Color for Lower, Middle & Upper Sections

### DISCUSSION THREAD

**Discussion by sameermdkhan** [xamarin/XamarinCommunityToolkit#1338] [Q&A]
Is it possible to have different "track" color for lower, middle & upper sections in Range Slider ? My requirement is to have lower track color as RED, middle one as YELLOW and upper as GREEN. 

Thanks,
Sameer

**Comment by AndreiMisiukevich** [comment]
Hi @sameermdkhan ) Interesting
For now it's impossible, but it definitely can be an enhancement
